### ISSUE TITLE
Convention Proposal Dump: 3-Hand Unnecessary Trash Compositions

### ISSUE THREAD

**Issue by pianoblook** [hanabi/hanabi.github.io#687]
A majority of these conventions build off the more nuanced Trash interpretations brought up in #311 and #363. Although I know those were denied back then, I figured I'd share this doc in its entirety to hopefully stir up discussion.

We *have* been moving in this direction, with the adoption of conventions like #614 and those proposed in #479. In any case,  I **strongly** believe these sorts of moves create huge amounts of space for creativity and gives a lot of options for maneuvering through potentially tricky situations.

Most of these are also largely theoretical. I'll be on the lookout for in-game opportunities moving forward.

Full doc here: https://docs.google.com/document/d/1hqZTnK90-0VR6biHrgmnGhZly9pKReVSCdTat-2-D5s/edit?usp=sharing

### Conventions Covered/Proposed:
> (** = fully compatible with current official conventions)
(*italics* = require chop-focused Unnecessary precedence, as proposed in #311)

**Unnecessary Trash Finesse Bad Trash Chop Move Ejection
**U. Trash Finesse Trash Push Ejection 
**U. Trash Finesse Trash Push Discharge
**U. Trash Finesse Playful Ignition
**U. UTD Trash Push Finesse
**U. UTD Bad Trash Chop Move Ejection
**U. UTD Trash Push Ejection
**U. UTD Push Discharge
**U. UTD Playful Ignition
*U. Bad Trash Chop Move Ejection Trash Push Finesse*
*U. Bad Trash Chop Move Ejection Trash Push Ejection*
*U. Bad Trash Chop Move Ejection Trash Push Discharge*
*U. Trash Push Discharge Trash Push Finesse*
*U. Trash Push Discharge Trash Push Ejection*
*U. Trash Push Discharge Trash Push Discharge*

TL;DR all the *Unknown Trash* versions work w/ conventions as-is, while the *Known Trash* versions build off of the assumption that their _Unnecessary_ forms turn into _Trash Pushes_ on Donald('s new _Chop_)

**Comment by pianoblook** [comment]
It's probably simpler to think of all of these as one giant proposal, which is *very* similar to @florian-5f3759df 's previous work from #614. 

Essentially it all boils down to proposing that the _Unnecessary_ component of a trash clue should *always* promise some reaction from the clued player's hand. 

A few notes:
- The _On-Chop Push_ interpretation has already been cemented, so I believe all of those should work as-is.
- If it's late in the game and one of the teammates can see that there was seemingly **no** other way to do a _Double Ignition_, DI should take precedence 
- As briefly mentioned in the doc, the _Playful Ignition_ forms should definitely be turned off if the team is low on clues or other niche situations where saving the playable seems important
